installers-conda/build_installers.py

```diff
@@ -304,6 +304,7 @@ def _definitions():
             "menuinst >=2.0.2",
             "mamba",
         ],
+        "exclude": ["pip"],
```

Comment:
Does the Spyder env include conda/mamba and/or micromamba? If so, please exclude them as well.

Reply:
`spyder_runtime` does not include `conda` or `mamba`. It has `spyder` as its only declared specification.